Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY go.mod ./
 
 RUN go mod download
 
-COPY server/main.go ./
+COPY main.go ./
```

Comment:
I don't think you want to do this change, we want the server binary to be in the docker container. Fix it, commit it and push the new commit to the branch.

Reply:
Server/main.go was moved to main.go in "root" directory.